### DISCUSSION TITLE
Wait for the user to finish typing something in before searching

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#116483] [Mobile]
Here the user hasn't even finished typing more than one little letter, and the mobile app just can't wait and insists on doing a useless search at this point. 

Can't the user be given a chance to type a little bit more first, or, like on a Google search, where he can even push the Enter button when he's ready?

![Screenshot_20240402_094030_GitHub.jpg](https://github.com/community/community/assets/1230959/1b86fe4f-db94-4ee8-8e4d-45a6a6deeff1)

Sure, you might say,
> Don't worry, just keep on typing. See? It all works out fine in the end. 

Well it's wasting precious bandwidth, and also precious CPU cycles back and GitHub.

> What are you, living back in 1945? All that isn't precious anymore.

Hey, I don't even have enough bandwidth to upload crystal clear screenshots. That's why they're all blurry. (Because I reduce their size in order to complete the upload during the limited time that GitHub needs before it fails.)

And, well, still, it makes the user nervous! He hasn't finished typing and then you go running off with the football.

> Look in your hands: the football still there. Don't bother that people around you are running.

Okay enough of that.
Yes, you are probably curious where exactly did I encounter this problem. Well it was when I was doing the actions in https://github.com/orgs/community/discussions/116482 .

**Comment by Iron-Ham** [comment]
Hi @jidanni,

Thanks for writing in! User feedback helps us improve the mobile experience. 
I understand that you'd like this search to be less active. At present, despite the spinner visibly showing on screen, we only begin to search when we detect no text inputs for over 250 ms. That said, I completely understand the concern, and I will file this as an internal issue with the Android team as a point of improvement.

Thank you for the feedback,
Hesham

**Reply by jidanni** [reply]
250ms? I see. The problem must be me. I still look around on the mobile keyboard for where the letters are. I'm not like one of those kids with the two thumbs busy.
